src/cpu/aarch64/acl_gemm_convolution.hpp

```diff
@@ -129,8 +141,8 @@ struct acl_gemm_convolution_fwd_t : public primitive_t {
             if (acp_.with_bias) {
                 auto scratchpad = scratchpad_registry().registrar();
                 const size_t bia_mem_sz_
-                        = acp_.bia_info.tensor_shape()[0] * sizeof(data_t);
-                scratchpad.template book<data_t>(
+                        = acp_.bia_info.tensor_shape()[0] * sizeof(bia_data_t);
```

Comment:
No need to `* sizeof(bia_data_t)` here because `book<bia_data_t>` will take this into account. See this comment https://github.com/oneapi-src/oneDNN/pull/886#discussion_r522527081

Reply:
@emfomenk , FYI I have updated this part in #886 (both in Winograd and GEMM convolutions), it works fine on the tests with `FWD_B`.

Reply:
I've also changed this as it wasn't too big a change.